### PR TITLE
WFP-1328

### PR DESCRIPTION
**Pull request by carlov20**

### CODE REVIEW

File: integration_tests/integration/allocate-cases-by-team.spec.ts [hunk 1]
```diff
@@ -1,278 +0,0 @@
...
-          },
-        ])
-    })
-
-    it('link to edit team list must exist', () => {
-      allocateCasesByTeamPage.link().should('contain', 'editing your team list')
-    })
-  })
-
-  context('Edge cases', () => {
```
Comment (sp-watson) [inline integration_tests/integration/allocate-cases-by-team.spec.ts:154]: Have these been removed? I think they are still useful tests.

File: integration_tests/integration/submit-select-teams.spec.ts [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+            code: 'TM2',
+            name: 'Team 2',
+          },
+          {
+            code: 'TM1',
+            name: 'Team 1',
+          },
+        ],
+      })
+      cy.task('stubPutUserPreferenceTeams')
```
Comment (sp-watson) [inline integration_tests/integration/submit-select-teams.spec.ts:56]: Is this verified anywhere? Not sure cypress detects whether stubs have been called

**Review by sp-watson — APPROVED** [review]
